app/brave_settings_strings.grdp

```diff
@@ -1703,7 +1703,7 @@ Leo AI. This action can't be undone.
     Allow Brave surveys
   </message>
   <message name="IDS_SETTINGS_SURVEY_PANELIST_DESC" desc="The description for settings option to enable survey panelist">
-    Allows Brave to show links to private surveys in the new tab page. Surveys help us build better features and measure the effectiveness of browser-based ad campaigns. <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>Learn more / see opt-out choices<ph name="END_LINK">&lt;/a&gt;</ph>
```

Comment:
Changed `Learn more` link label for Desktop, basing on this comment: https://github.com/brave/brave-core/pull/28995#discussion_r2127574397